content/learn/migration-guides/0.11-0.12/_index.md

```diff
@@ -1531,3 +1531,23 @@ App::new()
 ```
 
 </div>
+
+### [View Transformations](https://github.com/bevyengine/bevy/pull/9726)
+
+`mesh_functions::mesh_position_world_to_clip` was moved and renamed to `view_transformations::position_world_to_clip`. It now also takes a `vec3` instead of a `vec4` so you will need to use `vec4.xyz` to get a `vec3`.
+
+```rust
+// 0.11
+#import bevy_pbr::mesh_functions::mesh_position_world_to_clip
+fn mesh_position_local_to_clip(model: mat4x4<f32>, vertex_position: vec4<f32>) -> vec4<f32> {
+    let world_position = mesh_position_local_to_world(model, vertex_position);
+    return mesh_position_world_to_clip(world_position);
+}
+
+// 0.12
+#import bevy_pbr::view_transformations::position_world_to_clip;
+fn mesh_position_local_to_clip(model: mat4x4<f32>, vertex_position: vec4<f32>) -> vec4<f32> {

```

Comment:
This is incorrect. There is no `mesh_position_local_to_clip` function in `bevy_pbr::view_transformations`. None of the functions in `bevy_pbr::view_transformations` take a mat as a parameter.

Reply:
This is showing the migration of the `mesh_position_local_to_clip` function that uses `position_world_to_clip`. This isn't in `view_transformations` it was to show the import.

Reply:
it was copied from `mesh_functions.wgsl`

Reply:
Yeah okay, checks out.

I wonder if the function signature is just a bit of noise we don't need here though.

Reply:
I see what you mean now. I only very briefly looked at this. The similarity in names between everything here may also make things confusing in the guide. idk though

Reply:
I agree that the function signature is confusing and noise.

Reply:
Maybe just show in diff format like this:
```diff
-#import bevy_pbr::mesh_functions::mesh_position_world_to_clip
+#import bevy_pbr::view_transformations::position_world_to_clip;
 fn mesh_position_local_to_clip(model: mat4x4<f32>, vertex_position: vec4<f32>) -> vec4<f32> {
     let world_position = mesh_position_local_to_world(model, vertex_position);
-    return mesh_position_world_to_clip(world_position);
+    return position_world_to_clip(world_position.xyz);
 }
 ```